base-notebook/Dockerfile

```diff
@@ -105,8 +103,7 @@ WORKDIR /tmp
 
 # Prerequisites installation: conda, mamba, pip, tini
 RUN wget --quiet "https://github.com/conda-forge/miniforge/releases/download/${miniforge_version}/${miniforge_installer}" && \
-    echo "${miniforge_checksum} *${miniforge_installer}" | sha256sum --check && \
-    /bin/bash "${miniforge_installer}" -f -b -p "${CONDA_DIR}" && \
```

Comment:
Please, leave this line as it was, because it's safer from the bash perspective.

Reply:
ah my bad I didn't notice I dropped the quotes, thanks for catching that.